src/res/translation/translation_de_DE.ts

```diff
@@ -1085,12 +1085,12 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../../clientdlg.cpp" line="207"/>
         <source>If this LED indicator turns red, you will not have much fun using the %1 software.</source>
-        <translation type="unfinished"></translation>
+        <translation>Wenn diese LED rot leuchtet, dann wirst du keinen Spaß mit der %1 Software haben.</translation>
     </message>
     <message>
         <location filename="../../clientdlg.cpp" line="315"/>
         <source>O&amp;wn Fader First</source>
-        <translation type="unfinished"></translation>
+        <translation>Eigener Kanal zuerst (&amp;w)</translation>
```

Comment:
Hm, I think in other cases we just chose a useful accelerator letter in the translation, maybe we can do that here, too, to avoid the additional braces?

Reply:
```suggestion
        <translation>Eigener &amp;Kanal zuerst</translation>
```